worms_game.py

```diff
@@ -117,13 +133,29 @@ def _handle_events(self):
                     case pygame.K_LEFT | pygame.K_RIGHT:
                         self.current_worm.stop_moving()
 
+            if event.type == pygame.KEYDOWN:
+                if event.key == pygame.K_g:
+                    self.current_weapon = 'grenade'
+                    self.weapon_message = "Weapon: Grenade"
+                elif event.key == pygame.K_c:
+                    self.current_weapon = 'rocket'
+                    self.weapon_message = "Weapon: Rocket"
```

Comment:
Du coup ici ça donnerait plutôt ça 
```suggestion
            if event.type == pygame.KEYDOWN:
                if event.key == pygame.K_g:
                    self.current_worm.current_weapon = Grenade()
                elif event.key == pygame.K_c:
                    self.current_worm.current_weapon = Rocket()
```
Le weapon_message a pas besoin d'être stocké en tant qu'attribut de `Game`, on peut le déterminer en faisant "Weapon: " + self.current_worm.current_weapon.name